### DISCUSSION TITLE
Can I seek help from the community regarding the issues of my personal Machine Learning projects in this platform?

### DISCUSSION THREAD

**Discussion by MdEhsanulHaqueKanan** [community/community#146378] [Programming Help]
I have just joined this community. I was just wondering if I can discuss about my personal project issues here. Is it allowed to do in the platform?
I am doing a Machine Learning Flask app on Electric Vehicle Price prediction. I am enjoying working on it. But I am getting stuck on it time to time. So, I am looking for a helpful community.

**Comment by Stansbury-C** [comment]
For technical help I'd recommend Stack Overflow. You'll get a much better response there from the same people here!
I would also advise searching for your query first before just jumping in and adding a new question 
;-)

**Comment by tuves** [comment]
Thanks for posting in the GitHub Community, @MdEhsanulHaqueKanan! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
